lib/vanagon/platform.rb

```diff
@@ -226,7 +230,7 @@ def is_unix?
     #
     # @return [true, false] true if it is a windows variety, false otherwise
     def is_windows?
-      return !!@name.match(/^win-.*$/)
+      return !!@name.match(/^windows-.*$/)
```

Comment:
Double bang was pretty common in ruby books from circa 2007. I found that to be more idiomatic ruby than `!thing.nil?`. Also, in lock_manager, we totes disabled that check.